### PR TITLE
[T] (quests) creation

### PR DESCRIPTION
**Pull request by nai200**
Co-authored-by: xin_yi_hl <1012737146@qq.com>
Co-authored-by: 海福 <18917570841@163.com>
Co-authored-by: 寒焱炙冰 <realsunmengxiang@foxmail.com
>

### CODE REVIEW

File: config/ftbquests/quests/chapters/creation.snbt [hunk 1]
```diff
@@ -259,20 +259,20 @@
...
+			subtitle: "制造一个烈焰人燃烧室"
 			description: [
-				"While the Steam Engine works fueled by Campfires, it's &cnot very powerful&r. Campfires only provide for 2,048 stress units."
+				"当蒸汽引擎被篝火加热时，它并&c不能十分有力的运转&r。篝火驱动仅仅只能提供2048su的动力。"
 				""
-				"Instead of Campfires, I can use &bBlaze Burners&r to provide heat."
+				"我可以使用&b烈焰人燃烧室&r而不是篝火来提供热量。"
 				""
-				"A Blaze Burner is &cequivalent to a Campfire&r by default, but can be fed with furnace fuels to be &68 times stronger&r, producing 16,384su."
+				"在熄灭状态下，一个烈焰人燃烧室就&c相当于一个篝火&r，但是，当它被投喂可燃物后，效率将&增加68倍&r，达到16,384su。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/creation.snbt:273]:
```suggestion
				"在熄灭状态下，一个烈焰人燃烧室就&c相当于一个篝火&r，但是，当它被投喂可燃物后，效率将&增加68倍&r，达到16384su。"
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 2]
```diff
@@ -308,24 +308,23 @@
...
 			y: 11.5d
 			shape: "gear"
-			subtitle: "The sky is the limit"
+			subtitle: "天空即为极限"
 			description: [
-				"Now that I've unlocked rotational energy, I can &6create an unimaginable variety of contraptions&r to farm, kill, build, and even mine for me."
+				"现在，我已经掌握了旋转能，利用这股能量，我可以做到诸如：种植，建造，砍杀甚至采集矿物等不可思议的装置，这将取决于我的想象力。掌控它，我将在不同的方面得到帮助。"
 				""
-				"My research notes on mechanical energy ends here, but I have so many more ideas..."
+				"我对旋转能系统的研究笔记止步于此，但研究之路并未走到极限……."
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/creation.snbt:319]:
```suggestion
				"我对旋转能系统的研究笔记止步于此，但研究之路并未走到极限……"
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 3]
```diff
@@ -371,17 +370,15 @@
...
-			subtitle: "Make an Electric Motor"
-			description: [
-				"I don't know why I didn't think of this sooner. I can convert mechanical energy into electric energy, so &6why can't I do the reverse?&r"
-				""
-				"The &bElectric Motor&r will convert electricity into rotational force. With &6Tesseracts&r, I can now create contraptions without having to connect to my Steam Engines!"
+			subtitle: "慢着……"
+			description: ["我不明白为什么我没有早日想到这一点。我不止可以将机械中的旋转能转化为电能，也可以将电能转化为旋转能！在超立方体的帮助之下，我可以在任何地方创造便携式设备。这将是一种游戏方式上的转变。"]
+			dependencies: [
+				"080BD2AA7CCAAA90"
+				"6B7606A65BA8FE12"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/creation.snbt:380]: 这里是咋回事

Comment (nai200) [reply config/ftbquests/quests/chapters/creation.snbt:380]: 版本导致的？

File: config/ftbquests/quests/chapters/creation.snbt (hunk 1 above)
Comment (Ricky-fight) [reply config/ftbquests/quests/chapters/creation.snbt:273]:
```suggestion
				"在熄灭状态下，一个烈焰人燃烧室就&c相当于一个篝火&r，但是，当它被投喂可燃物后，&6效率将增加&68倍&r，达到16,384su。"
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 4]
```diff
@@ -221,13 +221,13 @@
...
-				"Now that I've had a taste of steam power, I've come to a conclusion;"
+				"现在我尝到了蒸汽动力的甜头，逐渐萌生出一个想法；"
 				""
-				"I need &4more&r. I need &belectricity&r."
+				"我要&4更多&r，我要&b电能&r。"
 				""
-				"To that end, the &bAlternator&r will &6convert RPM to electricity&r, measured in FE."
+				"怀揣着这个目的，我发现&b交流发电机&r将会把&6RPM转化为FE形式的电能&r。"
 				""
 				"&oUnlocks several quests in &5Innovation of Inconnu&r&o under &3Prometheus' Heresy&r"
```
[the remark this answers is not in the document]
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/creation.snbt:232]:
```suggestion
				"&oUnlocks several quests in &5Innovation of Inconnu&r&o under &3Prometheus' Heresy&r"
```

File: config/ftbquests/quests/chapters/creation.snbt (hunk 3 above)
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/creation.snbt:380]:
```suggestion
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 5]
```diff
@@ -371,17 +370,12 @@
...
 			y: 11.5d
 			shape: "square"
-			subtitle: "Make an Electric Motor"
-			description: [
-				"I don't know why I didn't think of this sooner. I can convert mechanical energy into electric energy, so &6why can't I do the reverse?&r"
-				""
-				"The &bElectric Motor&r will convert electricity into rotational force. With &6Tesseracts&r, I can now create contraptions without having to connect to my Steam Engines!"
+			subtitle: "慢着……"
+			description: ["我不明白为什么我没有早日想到这一点。我不止可以将机械中的旋转能转化为电能，也可以将电能转化为旋转能！在超立方体的帮助之下，我可以在任何地方创造便携式设备。这将是一种游戏方式上的转变。"]
 			]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/creation.snbt:378]:
```suggestion
			]
			dependencies: ["6B7606A65BA8FE12"]
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 6]
```diff
@@ -371,15 +370,11 @@
...
 		}
 		{
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/creation.snbt:372]:
```suggestion
		{
		title: "慢着…"
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 7]
```diff
@@ -371,15 +370,11 @@
...
 			x: 2.0d
 			y: 11.5d
 			shape: "square"
-			subtitle: "Make an Electric Motor"
-			description: [
-				"I don't know why I didn't think of this sooner. I can convert mechanical energy into electric energy, so &6why can't I do the reverse?&r"
-				""
-				"The &bElectric Motor&r will convert electricity into rotational force. With &6Tesseracts&r, I can now create contraptions without having to connect to my Steam Engines!"
+			subtitle: "慢着……"
+			description: ["我不明白为什么我没有早日想到这一点。我不止可以将机械中的旋转能转化为电能，也可以将电能转化为旋转能！在超立方体的帮助之下，我可以在任何地方创造便携式设备。这将是一种游戏方式上的转变。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/creation.snbt:377]: 重新翻吧
```suggestion
			description: [
				"I don't know why I didn't think of this sooner. I can convert mechanical energy into electric energy, so &6why can't I do the reverse?&r"
				""
				"The &bElectric Motor&r will convert electricity into rotational force. With &6Tesseracts&r, I can now create contraptions without having to connect to my Steam Engines!"
			]
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 8]
```diff
@@ -371,15 +370,11 @@
...
-			title: "Wait a Tick..."
 			x: 2.0d
 			y: 11.5d
 			shape: "square"
-			subtitle: "Make an Electric Motor"
-			description: [
-				"I don't know why I didn't think of this sooner. I can convert mechanical energy into electric energy, so &6why can't I do the reverse?&r"
-				""
-				"The &bElectric Motor&r will convert electricity into rotational force. With &6Tesseracts&r, I can now create contraptions without having to connect to my Steam Engines!"
+			subtitle: "慢着……"
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/creation.snbt:376]:
```suggestion
			subtitle: "Make an Electric Motor"
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 9]
```diff
@@ -371,15 +370,12 @@
 			]
 		}
 		{
-			title: "Wait a Tick..."
+		title: "慢着…"
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/creation.snbt:373]:
```suggestion
			title: "慢着…"
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 10]
```diff
@@ -371,15 +370,12 @@
 			]
 		}
 		{
-			title: "Wait a Tick..."
+			title: "慢着…"
 			x: 2.0d
 			y: 11.5d
 			shape: "square"
 			subtitle: "Make an Electric Motor"
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/creation.snbt:377]: 也要改

File: config/ftbquests/quests/chapters/creation.snbt (hunk 4 above)
Comment (xinyihl) [inline config/ftbquests/quests/chapters/creation.snbt:232]:
```suggestion
				"&o解锁位于&3普罗米修斯的异端&r&o之下的&5全新的革命&r&o任务"
```

File: config/ftbquests/quests/chapters/creation.snbt (hunk 10 above)
Comment (xinyihl) [inline config/ftbquests/quests/chapters/creation.snbt:377]:
```suggestion
			subtitle: "制作一个电动马达"
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 11]
```diff
@@ -371,15 +370,12 @@
...
-			description: [
-				"I don't know why I didn't think of this sooner. I can convert mechanical energy into electric energy, so &6why can't I do the reverse?&r"
-				""
-				"The &bElectric Motor&r will convert electricity into rotational force. With &6Tesseracts&r, I can now create contraptions without having to connect to my Steam Engines!"
+			description: ["我不明白为什么我没有早日想到这一点。我不止可以将机械中的旋转能转化为电能，也可以将电能转化为旋转能！在超立方体的帮助之下，我可以在任何地方创造便携式设备。这将是一种游戏方式上的转变。"]
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/creation.snbt:378]:
```suggestion
			description: [
				"我不明白为什么我没有早点想到这一点。我可以把机械能转化成电能所以&6为什么我不能反过来做呢?&r"
				""
				"&b电动马达&r将把电转化为机械能。在&6超立方体&r的帮助之下，我现在可以创造精巧的装置，而不必连接到我的蒸汽机！"]
```

File: config/ftbquests/quests/chapters/creation.snbt [hunk 12]
```diff
@@ -371,15 +370,15 @@
...
-			title: "Wait a Tick..."
+			title: "慢着…"
 			x: 2.0d
 			y: 11.5d
 			shape: "square"
-			subtitle: "Make an Electric Motor"
+			subtitle: "制作一个电动马达"
 			description: [
-				"I don't know why I didn't think of this sooner. I can convert mechanical energy into electric energy, so &6why can't I do the reverse?&r"
+				"我不明白为什么我没有早点想到这一点。我可以把机械能转化成电能所以&6为什么我不能反过来做呢?&r"
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/creation.snbt:379]:
```suggestion
				"我不明白为什么我没有早点想到这一点。我可以把机械能转化成电能所以&6为什么我不能反过来做呢？&r"
```

**Comment by Ricky-fight** [comment]
应该问题不大了？

**Pull request merged**